modules/home/assets/js/components/top-section.js

```diff
@@ -1,7 +1,44 @@
-const TopSection = () => {
+import { Box } from '@elementor/ui';
+import Typography from '@elementor/ui/Typography';
+import Button from '@elementor/ui/Button';
+import StyledYoutubeIcon from '../icons/youtube-icon';
+
+const TopSection = ( { ctaUrl, videoUrl, embedUrl } ) => {
 	return (
-		<p>Add the TopSection component here</p>
+		<Box sx={ { display: 'flex', flexDirection: { xs: 'column', md: 'row' }, backgroundColor: 'background.default', p: { xs: 1, md: 4 } } }>
+			<Box sx={ { pr: { xs: 0, md: 3 } } }>
+				<Box sx={ { display: 'flex', flexDirection: 'column', maxWidth: '600px', pb: 2 } }>
+					<Typography variant="body2">Hi!</Typography>
+					<Typography variant="h6">Welcome to Elementor</Typography>
+					<Typography variant="body2" >Get introduced to Elementor by watching our &ldquo;Getting Started&rdquo; video series. It will guide you through the steps needed to create your website. Then click to create your first page.</Typography>
+				</Box>
+				<Box sx={ { display: 'flex', gap: 1, pb: 1 } }>
```

Comment:
Isn't `gap` sufficient?

```suggestion
				<Box sx={ { display: 'flex', gap: 1 } }>
```

Reply:
no, gap defines gap between children inside the box, when they are in 1 line there is no space between buttons and the video, but I moved it 1 parent up.

Reply:
I would say: can we remove `pb` from here?
And on parent level add a `gap`?